aroma/tests/test_integration.py

```diff
@@ -54,26 +54,22 @@ def test_integration(skip_integration, nilearn_data):
     assert isfile(join(out_path, "melodic_IC_thr_MNI2mm.nii.gz"))
 
     # Check classification overview file
-    true_classification_overview = pd.read_csv(
-        join(resources_path, "classification_overview.txt"),
-        sep="\t",
-        index_col="IC",
-        nrows=4,
-    )
+    true_classification_overview = pd.read_csv(join(resources_path, "classification_overview.txt"),
+                                               sep="\t",
+                                               index_col="IC",
+                                               nrows=4,)
     classification_overview = pd.read_csv(
         join(out_path, "classification_overview.txt"), sep="\t", index_col="IC", nrows=4
     )
 
-    assert np.allclose(
-        true_classification_overview.iloc[:, 1:],
-        classification_overview.iloc[:, 1:],
-        atol=0.05,
-    )
+    assert np.allclose(true_classification_overview.iloc[:, :-1].values,
+                       classification_overview.iloc[:, :-1].values,
+                       atol=0.9)
 
     #  Check feature scores
-    f_scores = np.loadtxt(join(out_path, "feature_scores.txt"))
-    f_true = np.loadtxt(join(resources_path, "feature_scores.txt"))
-    assert np.allclose(f_true[0, :], f_scores[0, :], atol=0.01)
+    f_scores = pd.read_csv(join(out_path, "feature_scores.txt"), sep="\t")
+    f_true = pd.read_csv(join(resources_path, "feature_scores.txt"), sep="\t")
```

Comment:
You could use `read_table` instead and you wouldn't have to specify the separator.

Reply:
ohhh you wanted me to do the change on the test script?

Reply:
This is a minor thing, though, and we can hold off on it until another PR.